aspnet/web-api/overview/security/external-authentication-services.md

```diff
@@ -139,22 +134,23 @@ Once you have obtained your application ID and secret key, use the following ste
 <a id="GOOGLE"></a>
 ## Enabling Google Authentication
 
-Google is by far the easiest of the external authentication services to enable because it doesn't require a developer account, nor does it require additional information like your application ID or secret key as the other external authentication services necessitate.
+Using Google authentication requires you to create a Google developer account, and your project will require an application ID and secret key from Google in order to function. For information about creating a Google developer account and obtaining your application ID and secret key, see [https://developers.google.com](https://developers.google.com).
+
 
 To enable Google authentication for your web application, use the following steps:
 
-1. When your project is open in Visual Studio 2013, open the *Startup.Auth.cs* file:
+1. When your project is open in Visual Studio 2017, open the *Startup.Auth.cs* file:
```

Comment:
* The colon here presumes an image following, but the image reference is commented out.
* Why is the image ref commented rather than deleted if not needed?

These comments apply in a number of places.

Reply:
Ok, will do